runtime/onert/core/src/exec/ExecutorBase.h

```diff
@@ -53,6 +53,8 @@ class ExecutorBase : public IExecutor
    * @param tensor_builders Tensor builders that are currently used
```

Comment:
IMHO, some comment saying "if input_tensor or output_tensor is empty, that means... " could help reader.
Or 
maybe we need another ctor (without `input_tensor` or `output_tensor` param) for subgraph?